language/move-vm/runtime/src/native_functions.rs

```diff
@@ -24,6 +25,26 @@ pub type NativeFunction =
 
 pub type NativeFunctionTable = Vec<(AccountAddress, Identifier, Identifier, NativeFunction)>;
 
+pub fn make_table(
+    addr: AccountAddress,
+    elems: &[(&str, &str, NativeFunction)],
+) -> NativeFunctionTable {
+    elems
+        .iter()
+        .cloned()
+        .map(|(module_name, func_name, func)| {
+            (
+                addr,
+                Identifier::new(module_name).unwrap(),
+                Identifier::new(func_name).unwrap(),
+                func,
+            )
+        })
+        .collect()
+}
+
+pub type NoExecEnv = ();
```

Comment:
Hey @wrwg! Is this type ever used? Or is added to be used in the future?

Reply:
Could catch, removed.